packages/@divvi/mobile/src/public/index.ts

```diff
@@ -5,6 +5,7 @@
  * - Enable better tree-shaking
  * - Prevent accidental exposure of internal implementation details
  */
+export { BtnSizes, BtnTypes, Button, TextSizes } from './components/Button'
```

Comment:
We should export types with the `type` identifier for clarity and explicitness.

```suggestion
export { type ButtonSize, type ButtonType, Button, type ButtonTextSize } from './components/Button'
```